src/ProgressBars.jl

```diff
@@ -48,18 +48,20 @@ mutable struct ProgressBar
   description::AbstractString
   multilinepostfix::AbstractString
   mutex::Threads.SpinLock
+  output_stream::IO
 
   function ProgressBar(wrapped::Any;
                        total::Int=-2,
                        width::Union{UInt, Nothing}=nothing,
                        leave::Bool=true,
                        unit::AbstractString="",
                        unit_scale::Bool=true,
-                       printing_delay::Number=0.05)
+                       printing_delay::Number=0.05,
+                       output_stream::IO=stderr)
     this = new()
     this.wrapped = wrapped
     if width == nothing
-        this.width = displaysize(stdout)[2]
+        this.width = displaysize(stderr)[2]
```

Comment:
Here we should use `displaysize(output_stream)`. This might make a difference if e.g. stdout is piped into a file.